### DISCUSSION TITLE
Is this a bug? No apps shown / found

### DISCUSSION THREAD

**Discussion by tarasis** [will-stone/browserosaurus#542] [Q&A] [answered]
First time installing the app, I have chosen to let Browserosaurus be the default browser. 

Find Apps says 0 compatible apps found. I press "rescan". Switching to the "Apps" tab, I get a spinning progress wheel.

I have Firefox, Firefox Dev, Safari, Safari Tech Preview, Chrome, Edge, Opera, Waterfox Current all installed.

Also the preferences window disappears when I switch desktops.

According to activity monitor, the app is using 10-20% CPU. (Renderer and GPU processes)

I'm on a M1 Mac Book Air, running macOS 12.4. Its a fairly full 1TB disk.

**Comment by will-stone** [comment, marked as answer]
Yeah sorry about this. It's a known issue: https://github.com/will-stone/browserosaurus/issues/539 Please install 18.1 until I can put out a fix.
